test/perf.test.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:recase/recase.dart';
```

Comment:
Can you move this into `/tools`? Since it's not a unit test and would be kicked off manually. The PR should be good afterwards.

Reply:
Done!